cargo-dist/templates/installer/installer.sh.j2

```diff
@@ -21,7 +21,22 @@ APP_VERSION="{{ app_version }}"
 ARTIFACT_DOWNLOAD_URL="${INSTALLER_DOWNLOAD_URL:-{{ base_url }}}"
 PRINT_VERBOSE=${INSTALLER_PRINT_VERBOSE:-0}
 PRINT_QUIET=${INSTALLER_PRINT_QUIET:-0}
-NO_MODIFY_PATH=${INSTALLER_NO_MODIFY_PATH:-0}
+if [ -n "{{ '${' }}{{ no_modify_path_env_var }}:-}" ]; then
+    NO_MODIFY_PATH="${{ no_modify_path_env_var }}"
+else
+    NO_MODIFY_PATH=${INSTALLER_NO_MODIFY_PATH:-0}
+fi
+if [ "{{ '${' }}{{ disable_update_env_var }}:-0}" = "1" ]; then
+    INSTALL_UPDATER=0
+else
+    INSTALL_UPDATER=1
+fi
+UNMANAGED_INSTALL="{{ '${' }}{{ unmanaged_dir_env_var }}:-}"
+if [ -n "${UNMANAGED_INSTALL}" ]; then
+    NO_MODIFY_PATH=1
+    INSTALL_UPDATER=0
+fi
```

Comment:
ditto on mode should go first..?

Reply:
Honestly I think "unmanaged mode" is opinionated enough that it probably should override the individual options instead of the other way around. I see the individual options as being useful in normal mode, not for this.